website/src/components/MenuTitleChildren/MenuTitleChildren.jsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react"
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import HoverVisible from "../hoverVisible/HoverVisible";
+import './MenuTitleChildren.scss';
+
+const RenderChild = (props) => {
+    const [isVisible, setIsVisible] = useState(false)
+    const selectedField = useSelector((state) => state.treeData.selectedField);
+    
+    return(
+            <div 
+                onMouseEnter={() => setIsVisible(true)}
+                onMouseLeave={() => setIsVisible(false)}
+                onMouseOver={() => setIsVisible(true)}
+                key={props.index} 
+                className={`child ${selectedField === props.path ? 'selected' : ''}`} 
+                onClick={() => props.onClick(props.path)}
+            >
+                <div className="child-name">
+                    {props.name}
+                </div>
+                {
+                    props.showDelete ?
+                        <HoverVisible isVisible={isVisible}>
+                            <div className="delete-img" onClick={props.onDeleteClick.bind(this, props.path)}>
+                                <i className="fa-solid fa-trash-can fa-sm"></i>
+                            </div>
+                        </HoverVisible>
+                    : ''
+                }
+                
+            </div>
+        
+    )
+}
+
+const MenuTitleChildren = (props) => {
+    const children = []
+    useEffect(()=>{}, [props.children])
+    props.children.forEach((child, index) => {
```

Comment:
in action, `props.children` is any content passed in the body. any inner component. 
if we pass to it an array of JSON items. it's better to dedicate a name to it. like `items`